http/src/routing.rs

```diff
@@ -701,6 +703,15 @@ pub enum Route {
         /// The ID of the user.
         user_id: u64,
     },
+    /// Route information to search for members in a guild.
+    SearchGuildMembers {
+        /// The ID of the guild.
```

Comment:
Don't use "The" here, otherwise, every docstring would start with "The".

Reply:
Removed them.